packages/ddp-client/common/livedata_connection.js

```diff
@@ -797,7 +798,7 @@ export class Connection {
         callback = (err) => {
           err && Meteor._debug("Error invoking Method '" + name + "'", err);
         };
-      } else {
+      } else if (Meteor.isServer) {
```

Comment:
This is supposed to be created in the client as well. This is most likely not the solution. And some tests are failing because of this change.